core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1365,6 +1365,16 @@ object GroupMetadataManager {
 
   def maybeConvertOffsetCommitError(error: Errors) : Errors = {
     error match {
+      case Errors.NETWORK_EXCEPTION =>
+        // When committing offsets transactionally, we now verify the transaction with the
+        // transaction coordinator. Verification can fail with `NETWORK_EXCEPTION`, a retriable
+        // error which older clients may not expect and retry correctly. We have the option of
+        // translating `NETWORK_EXCEPTION` to either `COORDINATOR_LOAD_IN_PROGRESS` or
```

Comment:
Thanks for the explanation. I wonder if we could leave out the sentence about "we have the option..." and just include " We use `COORDINATOR_LOAD_IN_PROGRESS` because it retries the request without an unnecessary coordinator lookup."

Reply:
Thanks, I've reworded the comments.